_gsocproposals/2022/proposal_CernVM-FS-firecracker-based-integration-tests.md

```diff
@@ -0,0 +1,57 @@
+---
+title: CernVM-FS integration test framework powered by Firecracker VM
+layout: gsoc_proposal
+project: CernVM
```

Comment:
You have to add a `CernVM` project in `_gsocprojects` directory. (But last year, you had a `CernVM-FS` project !)

Reply:
My mistake, it should be CernVM-FS. I copied the project file from 2021.